plugins/tableselection/plugin.js

```diff
@@ -761,7 +761,7 @@
 							targetNode = targetNode.getPreviousSourceNode( false, CKEDITOR.NODE_ELEMENT, boundaryTable );
 						}
 
-						if ( !targetNode && !endNode.is( 'table' ) && endNode.getNext() ) {
+						if ( !targetNode && endNode && endNode.is && !endNode.is( 'table' ) && endNode.getNext() ) {
```

Comment:
When I ran all the tests, this fragment was failing for one TC (don't remember which one exactly), the thing was that endNode was a textNode - so it did not have `is` method. I decided to add it as a just extra safety.